Adyen/Components/Base/AddressInfo.swift

```diff
@@ -81,3 +82,21 @@ public struct AddressInfo: Equatable, Encodable {
     }
 
 }
+
+extension AddressInfo {
+    
+    /// :nodoc:
+    public var formatted: String {
```

Comment:
it would be nice to have a protocol `Formattable`, because we have the `formatted` property also in `Amount`, but please don't do it in this PR.